src/main/java/nl/tudelft/controller/collision/CollisionHelper.java

```diff
@@ -29,7 +29,8 @@
             throw new IllegalArgumentException();
         } else {
             List<AbstractGameObject> returnObjects = new ArrayList<>();
-            quad.retrieve(returnObjects, objectA.getBounds());
+
+            returnObjects = quad.retrieve(returnObjects, objectA.getBounds());
```

Comment:
I think it's save to set this. The game still works fine, but could someone with more experience have a quick look?